shopping_list_frontend/lib/services/notifiers/popups.dart

```diff
@@ -5,16 +5,25 @@ enum SnackBarPopUpType { error, info }
 class SnackBarPopUp {
   String? message;
   BuildContext context;
+  GlobalKey<ScaffoldMessengerState>? rootScaffoldMessengerKey;
 
-  SnackBarPopUp.message({required this.message, required this.context}) {
```

Comment:
don't forget it also can be as factory, pls fix it
Read [this](https://dart.dev/language/constructors#:~:text=Factory%20constructors-,%23,-When%20encountering%20one)